src/sentry/conf/server.py

```diff
@@ -3302,6 +3302,7 @@ def build_cdc_postgres_init_db_volume(settings: Any) -> dict[str, dict[str, str]
         "filters": {"filetypes": ["pe", "pdb"]},
         "url": "https://driver-symbols.nvidia.com/",
         "is_public": True,
+        "accept_invalid_certs": True,
```

Comment:
Can we add a TODO here?

Reply:
Good idea. What do you think it should say? That we want to remove it again when the underlying problem is fixed?

Reply:
Yeah something like that. Maybe a definition of why we had this issue in the first place. In the future the context might be lost and we might have to dig deep into Github in order to find the root cause.

Reply:
`once the cert problem is cleared up` meaning when CURL supports XXX certification feature. might be a better description with a more searchable comment

Reply:
I made the comment more specific. @Swatinem to your knowledge, is this an accurate description of the problem?